client-tools/Shell/ShellFeature.h

```diff
@@ -63,7 +90,11 @@ class ShellFeature final : public ArangoshFeature {
   std::vector<std::string> _positionals;
   std::string _unitTestFilter;
   std::vector<std::string> _scriptParameters;
+  std::unique_ptr<TelemetricsHandler> _telemetricsHandler;
   bool _runMain{false};
+#ifdef ARANGODB_ENABLE_MAINTAINER_MODE
```

Comment:
Here, the member is made available if `ARANGODB_ENABLE_MAINTAINER_MODE`, but above (https://github.com/arangodb/arangodb/pull/18174/files#diff-e43190b071f8b4c4829e9e3080f74b3481e1a8d06f9f14ceada0485931022dfcR59) the member is modified if `ARANGODB_ENABLE_FAILURE_TESTS`. 
This is inconsistent.